jgclark.Summaries/src/progress.js

```diff
@@ -128,6 +128,7 @@ export async function makeProgressUpdate(paramsIn: any = '', source: string = 'c
         GOMentionsTotal: paramProgressMentionsTotal,
         GOMentionsAverage: paramProgressMentionsAverage,
         GOMentionsExclude: [],
+        GOChecklistRefNote: config.progressChecklistReferenceNote,
```

Comment:
This doesn't look ideal, as it breaks the clear pattern above. 
What you want is to add the ability to pass this in as a parameter, like all the others, unless there's a good reason it doesn't make sense.